v4/source/introduction.yaml

```diff
@@ -214,23 +214,205 @@ tags:
 
       The list of Mattermost WebSocket events are:
       - added_to_team
+        ```
+        {
+          event: 'added_to_team',
+          data: {
+            team_id: '<team_id>',
+            user_id: '<user_id>'
+          },
+          broadcast: {
+            omit_users: null,
+            user_id: '<user_id>',
+            channel_id: '<channel_id>',
+            team_id: '<team_id>'
+          },
+          seq: '<seq_id>'
+        }
+        ```
       - authentication_challenge
-      - channel_converted
       - channel_created
+        ```
+        {
+          'event': 'channel_created',
+          'seq'  : '<seq_id>',
+          'data' : {
+            'channel_id': '<channel_id>',
+            'team_id'   : '<team_id>'
+          },
+          'broadcast': {
+            'omit_users': None,
```

Comment:
Can we be consistent for the 'omit_users' field as well? I've seen it as `null`, `None` and `'null'`

Reply:
`null` is the correct value, 8256c08.